.gitignore

```diff
@@ -1,2 +1,6 @@
 target
 project/project
+.idea
```

Comment:
Its usually not considered good practice to put IDE specific settings into `.gitignore`, rather you should be using a global gitignore.